app/src/marketplace/containers/EditProductPage/EditControllerProvider.jsx

```diff
@@ -84,10 +84,15 @@ function useEditController(product: Product) {
 
     const redirectToProductList = useCallback(() => {
         if (!isMounted()) { return }
-        history.replace(routes.products.index())
+        if (isDataUnionProduct(product)) {
```

Comment:
I would maybe leave it as it is... Editing a product returns to products 🤔

Reply:
True, probably more intuitive!

Reply:
If we are only interested in one event, we could just define a partial abi like in the client: https://github.com/streamr-dev/streamr-client-javascript/blob/21d668d228c560a0569908c19329220f0235b2a7/src/dataunion/Contracts.ts#L181

Reply:
woops sorry @tumppi accidentally deleted your comment 😬

Reply:
No worries 😄